### DISCUSSION TITLE
Example of writing a bigbed

### DISCUSSION THREAD

**Discussion by mrvollger** [jackh726/bigtools#7] [Q&A]
Hello,

I was excited to come across this crate, thanks for making it! Do you have an example of writing a bigbed file from bed data (ideally bed12 data)?

thanks!
Mitchell

**Comment by jackh726** [comment]
Hi @mrvollger, sorry I missed this.

Probably the best example is the `bedtobigbed` binary code: https://github.com/jackh726/bigtools/blob/master/src/bin/bedtobigbed.rs

Let me know if this is unclear, or if you have other questions.

**Comment by jackh726** [comment]
I'm going to convert this to a discussion, since this isn't a particular actionable item (but please feel free to continue asking questions)
